sklearn/impute/_base.py

```diff
@@ -597,8 +597,10 @@ def fit(self, X, y=None):
 
         Returns
         -------
-        self : object
-            Returns self.
+        Xt : {ndarray or sparse matrix}, shape (n_samples, n_features)
+            The missing indicator for input data. The data type of ``Xt``
+            will be boolean.
```

Comment:
You can take use the docstring from `_get_missing_features_info`, since `_fit` returns one of the outputs of `_get_missing_features_info`:

```py
        imputer_mask : {ndarray or sparse matrix}, shape \
(n_samples, n_features) or (n_samples, n_features_with_missing)
            The imputer mask of the original data.
```

Reply:
Sure. 

Thanks.

Reply:
Yup I agree with just `(n_samples, n_features)`.